LEGO1/mxdsfile.cpp

```diff
@@ -47,13 +47,13 @@ MxLong MxDSFile::Open(MxULong uStyle)
 }
 
 // OFFSET: LEGO1 0x100cc780
-MxLong MxDSFile::Read(unsigned char *pch, MxULong cch)
+MxResult MxDSFile::Read(unsigned char *p_buf, MxULong p_nbytes)
 {
-  if (m_io.Read((char*)pch, cch) != cch)
-    return -1;
+  if (m_io.Read(p_buf, p_nbytes) != p_nbytes)
+    return FAILURE;
 
-  m_position += cch;
-  return 0;
+  m_position += p_nbytes;
+  return SUCCESS;
```

Comment:
Maybe some of the other functions in `MxDSFile` can also be changed to use the result constants and `MxResult` instead of `MxLong`?

Reply:
I thought about doing that before, and I just looked again and remembered why I didn't. It's this line in Open():
https://github.com/isledecomp/isle/blob/830115044d18fd00ed33a4b821cc8b286c8a055e/LEGO1/mxdsfile.cpp#L25
`SUCCESS + 1`?

Reply:
Yeah, I think that's the only function in the class that seems to use a value that isn't `SUCCESS` or `FAILURE`. `ReadChunk` and `Close` use only those known values. I guess we should leave it as-is for now.